charts/policy-reporter/charts/ui/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels: {{ include "policyreporter.labels" . | nindent 4 }}
+  name: {{ include "policyreporter.fullname" . }}
+spec:
+  podSelector:
+    matchLabels: {{- include "ui.selectorLabels" . | nindent 6 }}
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    ports:
+    - protocol: TCP
+      port: {{ .Values.service.port }} 
+  egress:
+  - to:
+    ports:
+    - protocol: TCP
+      port: {{ .Values.service.port }} 
```

Comment:
Hey, it would be Great if you could add an additional veritable to be able add egress rules. This would be required if you add additional targets and have to access them

Reply:
You may only use the Policy Reporter UI but other users want to use the Loki Target with the NetworkPolicy and then they have to configure the egress to send informations to Loki, so they should have the ability to add this egress rule in the policy-reporter network policy